cohorts/load.py

```diff
@@ -1114,7 +1135,18 @@ def plot_boolean(self, on, boolean_col, col=None, ax=None,
         plot_col, df = self.as_dataframe(on, col, **kwargs)
         df = filter_not_null(df, boolean_col)
         df = filter_not_null(df, plot_col)
-        df[boolean_col] = df[boolean_col].astype(bool)
+
+        if boolean_label:
+            df[boolean_label] = df[boolean_col]
+            boolean_col = boolean_label
+
+        condition_value = None
+        if boolean_value_map:
+            assert (True in boolean_value_map) and (False in boolean_value_map), \
```

Comment:
Since you also want these to be the only keys, maybe `assert set(boolean_value_map.keys()) == set([True, False])`?